apps/frontend/src/components/NeighborhoodSearch/NeighborhoodSearch.tsx

```diff
@@ -0,0 +1,172 @@
+import { useEffect, useState } from 'react';
+import { Form, Container, Row, Col } from 'react-bootstrap';
+import { Neighborhood, NeighborhoodsPerPage } from '@neighborhood/backend/src/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { AxiosError } from 'axios';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import styles from './NeighborhoodSearch.module.css';
+import NeighborhoodCard from '../NeighborhoodCard/NeighborhoodCard';
+import CustomBtn from '../CustomBtn/CustomBtn';
+import CreateNeighborhoodModal from '../CreateNeighborhoodModal/CreateNeighborhoodModal';
+import neighborhoodsService from '../../services/neighborhoods';
+import SpinWheel from '../SpinWheel/SpinWheel';
+import AlertBox from '../AlertBox/AlertBox';
+import { ErrorObj } from '../../types';
+
+export default function NeighborhoodSearch({
+  neighborhoods,
+  cursor,
+  isNextPage,
+}: {
+  neighborhoods: Neighborhood[];
+  cursor?: number;
+  isNextPage: boolean;
+}) {
+  const [neighborhoodList, setNeighborhoodList] = useState(neighborhoods);
+  const [hasNextPage, setHasNextPage] = useState(isNextPage);
+  const [currentCursor, setCurrentCursor] = useState(cursor);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [show, setShow] = useState(false);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const fetchData = async function fetchData() {
+    setErrorMsg('');
+    setIsLoading(true);
+
+    try {
+      const data = (await neighborhoodsService.getNeighborhoods(
+        currentCursor,
+      )) as unknown as NeighborhoodsPerPage;
+
+      setNeighborhoodList(neighborhoodList.concat(data.neighborhoods));
+      setCurrentCursor(data.newCursor);
+      setHasNextPage(data.hasNextPage);
+    } catch (error) {
+      if (error instanceof AxiosError) {
+        const errorResponse: ErrorObj = error.response?.data;
+        setErrorMsg(errorResponse.error);
+      } else if (error instanceof Error) {
+        setErrorMsg(error.message);
+      }
+
+      window.scrollTo(0, 0);
+      console.error(error);
```

Comment:
I don't think this is needed anymore

Reply:
Are you talking about the `console.error`? I guess not

Reply:
Yep, the console error